i18n-util/webpack.config.js

```diff
@@ -0,0 +1,23 @@
+const path = require('path');
+const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+
+module.exports = {
+  mode: "development",
```

Comment:
Do we intend to build both dev/prod versions of this? If not - let's rely on production here to avoid the **development** warning generated in `i18n-util.bundle.f37aad80ce754dde7ffe.js`.

Reply:
created a new cmd for dev mode